src/install_update.ts

```diff
@@ -120,8 +127,10 @@ export async function updateDependencyFile(
 
     return updateSummary;
   } catch (err) {
-    throw new Error(err);
+    if (!(err.cause instanceof Deno.errors.NotFound)) throw err;
```

Comment:
Worth noting, as this logic is also found elsewhere. Sanity check on the below appreciated:

If supported dependency file was not found, do not report as an error. The reason for this is that there may be any combination of supported dependency files used. The only case where bubbling this up would apply is if *no* dependency files were found, which, since `slack.json` is both necessary for the project AND a dependency file, is not a valid use case.

Reply:
Yeah, I was thinking about this as well and I think you're right. Whether or not a dev ends up using an `import_map` file is kind of an implementation detail and if they don't have a `slack.json` they won't get here at all 🤷 

I think this makes sense